src/Confluent.Kafka/Producer.cs

```diff
@@ -246,8 +247,8 @@ public void HandleDeliveryReport(Message message)
         ///     Initializes a new Producer instance.
         /// </summary>
         /// <param name="config">
-        ///     librdkafka configuration parameters (refer to https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md)
-        ///     topic configuration parameters should be specified via the "default.topic.config" config parameter.
+        ///     librdkafka configuration parameters (refer to https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md).
+        ///     Topic configuration parameters should be specified via the "default.topic.config" config parameter.
```

Comment:
"must be" or "are", "not should be".
Also clarify the default.topic.config, something like:
"via the default.topi... sub-dictionary"